pom.xml

```diff
@@ -154,13 +154,13 @@ SOFTWARE.
     <dependency>
       <groupId>org.hamcrest</groupId>
       <artifactId>hamcrest-core</artifactId>
-      <version>3.0</version>
+      <version>1.3</version>
```

Comment:
@VladMilash I still don't get it: why decreasing the version number? What's wrong with 3.0?